### PR TITLE
PEER-776 Update Post Rules

### PR DESCRIPTION
**Pull request by almondy22**

### CODE REVIEW

File: app/components/Base/TipsBase.js [hunk 1]
```diff
@@ -37,3 +37,13 @@ export default BaseRoundedNoPadding.extend`
     }
   }
 `;
+
+export const TipsBaseSmallPadding = TipsBase.extend`
```
Comment (Lampardks) [inline app/components/Base/TipsBase.js:41]: use emotion css and separate the components into different files

File: app/components/QuestionForm/PostRules.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  'common.contentPopupBlock_7',
+];
+
+const PostRules = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      css={css`
+        margin-bottom: 32px;
```
Comment (Lampardks) [inline app/components/QuestionForm/PostRules.jsx:27]: put all styles in a separate object and use the properties of this object in the component

File: app/components/Base/TipsBase.js (hunk 1 above)
Comment (almondy22) [reply app/components/Base/TipsBase.js]: Done

File: app/components/QuestionForm/PostRules.jsx (hunk 2 above)
Comment (almondy22) [reply app/components/QuestionForm/PostRules.jsx]: Done

**Review by Lampardks — APPROVED** [review]

**Pull request merged**